control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -5798,6 +5798,26 @@ def test_service_phase21_restore_rejects_non_mapping_payload_fields(self) -> Non
             restored_service.restore_authoritative_record_chain_backup(backup)
         self._assert_authoritative_store_empty(restored_store)
 
+    def test_service_phase21_restore_rejects_null_mapping_payload_fields(self) -> None:
+        _store, service, _promoted_case, _evidence_id, _reviewed_at = (
+            self._build_phase19_in_scope_case()
+        )
+        backup = service.export_authoritative_record_chain_backup()
+        backup["record_families"]["reconciliation"][0]["subject_linkage"] = None
+
+        restored_store, _ = make_store()
```

Ruff (0.15.12)

[error] Undefined name `make_store`

(F821)